### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by akrmn** [firebase/FirebaseUI-Android#77]
The Login fragment has hardcoded strings, which makes it difficult to use it in non-English projects.

**Comment by abehaskins** [comment]
Great point! We'll improve this for the next release.

**Comment by cardenuto** [comment]
@akrmn  I have been working on code that uses this library I needed to override the fragment_firebase_login.xml file locally and that gave me the ability to use resource strings instead. I am not sure this will help you in the short run since it has been a while. 

Feel free to use the code for other features 
https://github.com/cardenuto/FirebaseLogin any suggestions are welcome. Currently I designed my code to be added manually to a project as more of a template ... instructions are included in the repo.
